### PR TITLE
[DO NOT REBASE/REAPPLY]Nice ffmpeg to 15 and 2

### PR DESCRIPTION
**Pull request by Jorropo**
niceness is -20 to 20 little = execute prioritoraly (20 is near never execute, but don't execute faster if < 0 just steal other app cpu power)

Niceness 15 for video conversion
Niceness 2 for image generation (niceness 2 doesn't make it slower, just don't block servers and other app).

### CODE REVIEW

**Comment by Jorropo** [comment]
@rigelk made

File: server/helpers/ffmpeg-utils.ts [hunk 1]
```diff
@@ -56,6 +56,7 @@ async function generateImageFromVideoFile (fromPath: string, folder: string, ima
   try {
     await new Promise<string>((res, rej) => {
       ffmpeg(fromPath)
+        .renice(2)
```
Comment (Chocobozzz) [inline server/helpers/ffmpeg-utils.ts]: Please put this value in a constant in `constants.ts`

File: server/helpers/ffmpeg-utils.ts [hunk 2]
```diff
@@ -84,6 +85,7 @@ type TranscodeOptions = {
 function transcode (options: TranscodeOptions) {
   return new Promise<void>(async (res, rej) => {
     let command = ffmpeg(options.inputPath)
+                    .renice(15)
```
Comment (Chocobozzz) [inline server/helpers/ffmpeg-utils.ts]: Same here

**Comment by Jorropo** [comment]
@Chocobozzz is it right ?

**Comment by Jorropo** [comment]
@rigelk I made a pr in your repo containing this code :
[github.com/rigelk/PeerTube/pull/3](https://github.com/rigelk/PeerTube/pull/3)
You just have to merge it (whitout rebase !!!) and you will not have any conflict.

And @Chocobozzz don't rebase this for avoid conflict with rigelk:ffmpeg-presets.

**Comment by Jorropo** [comment]
Included in #679

**Pull request closed without merging**